css/index.css

```diff
@@ -0,0 +1,278 @@
+
+    
+
+body {
+    margin: 0;
+    font-family:  "Montserrat", sans-serif;
+    background-color:#F3F2EF;
+    color: #595151; /* Default text color */
+
+}
+
+nav {
+    display: flex;
+    justify-content: center; 
+    align-items: center;
+    background-color:#F3F2EF;
+    border-bottom: 1px solid #C7C7C7;
+    height: 80px;
+    padding: 0 20px;
+
+   
+}
+
+.nav-links a {
+    color:black;
+    text-decoration: none;
+    margin: 0 15px;
+    font-size: small;
+    padding: 10px; 
+}
+
+.nav-links a:hover {
+    color: #FF6400;
+    border-bottom: 1px solid #FF6400;
+}
+
+.nav-icons i {
+    color: grey;
+    margin: 0 10px;
+    font-size: 20px;
+    transition: color 0.3s ease;
+}
+
+.nav-icons i:hover {
+    color: #945DD6;
+}
+.nav-icons{
+    margin-right: 0px;
+}
+
+
+h1,h2{
+    color:rgb(8, 6, 6);
+    font-size:x-large;
+    font-weight: 500;
+  
+
+}
```

Comment:
**Suggestion:** Using the exact same styles for both the `h1` and `h2` might not be great. Since they are different levels of headings, their styles should reflect the difference in importance. The `h1` is usually the heading that stands out the most in the page. You could make it bigger, for instance.

Reply:
Got it. Updated.